proto/axelar/axelarnet/v1beta1/types.proto

```diff
@@ -56,4 +56,7 @@ message Fee {
   cosmos.base.v1beta1.Coin amount = 1 [ (gogoproto.nullable) = false ];
   bytes recipient = 2 [ (gogoproto.casttype) =
                             "github.com/cosmos/cosmos-sdk/types.AccAddress" ];
+  bytes refund_recipient = 3
```

Comment:
With cosmwasm, this will be exposed at the app layer too and could be involved in a multi-hop. Maybe we should keep this as a string too for consistency?

Reply:
There are no hooks in consensus that allows for doing multi-hop with axelarnet CallContract. With cosmwasm, smart contracts on Axelar can receive GMP calls from other chains and then trigger CallContract. Since that will be feasible, I'm suggesting to use a `string` type here

Reply:
if we need to support crosschain addresses, the address string is not enough to identify it, the chain is needed, too. I'd suggest the following: As long as this is only an internal object keep as is, once we know what exactly the requirements from cosmwasm contracts are we can remove the field and add a new one with an updated type

Reply:
You are absolutely right. The address itself isn't enough for us to do the refund. It's more true for those GMPs via IBC since over there it can be a multi-hop and we currently have no idea where the original chain is where the original request is initiated.

Reply:
Ok, yeah, since multi-hop is only possible here when cosmwasm will be added, we can add a separate field at that time if needed.